pkgdb2/templates/package.html

```diff
@@ -68,15 +68,15 @@
         <td colspan="2" align="center">
           <div id="monitoring_block">
             <input type="radio" id="no_monitoring" name="monitoring" value="0" />
-                <label for="no_monitoring" {%-
+                <label for="no_monitoring" title="Will toggle the monitor flag to monitor package for new releases" {%-
```

Comment:
What about something more direct, like: `Turns off monitoring new releases for this package`